internal/provider/client_mock_workspaces_test.go

```diff
@@ -200,3 +200,7 @@ func (m *mockWorkspaces) SetDataRetentionPolicyDontDelete(ctx context.Context, o
 func (m *mockWorkspaces) ListTagBindings(ctx context.Context, workspaceID string) ([]*tfe.TagBinding, error) {
 	panic("not implemented")
 }
+
+func (m *mockWorkspaces) AddTagBindings(ctx context.Context, workspaceID string, options tfe.WorkspaceAddTagBindingsOptions) ([]*tfe.TagBinding, error) {
+	panic("not implemented")
+}
```

Comment:
Wondering why this is here, was something failing without it? Don't see it's usage anywhere.

Reply:
This [requirement](https://github.com/hashicorp/go-tfe/commit/a8b7788fed459e82583448b5082ebaf6c665fb2e#diff-930187f85f6cf4c1050f29b94333154dfb9ba5dceb2750d2eba5be43135c34c9R139) is coming from go-tfe version 1.7.0. When I created this PR, the go-tfe version was still at 1.6.7. However, my changes will need at least 1.6.9, so I bumped the go-tfe version to 1.7.0 altogether. In reality I could have kept the go-tfe version as is, since the latest commit in main already bumped the go-tfe version to 1.6.9, which is the minimum version this change depends on.

Reply:
Ah gotcha!